data/model/user.py

```diff
@@ -1271,6 +1272,13 @@ def delete_user(user, queues):
     # Delete non-repository related items.
     _delete_user_linked_data(user)
 
+    # Delete the namespace size if it exists
+    with db_transaction():
+        try:
+            QuotaNamespaceSize.delete().where(QuotaNamespaceSize.namespace_user == user).execute()
+        except QuotaNamespaceSize.DoesNotExist:
+            pass
```

Comment:
Can you move this into `_delete_user_linked_data`. Also, `RepositorySize`s need to be cleaned up too?

Reply:
Since namespaces are unique, I don't think it's necessary to wrap this in a transaction.